.github/workflows/tests.yml

```diff
@@ -159,6 +159,25 @@ jobs:
             exit "0"
           fi
 
+  snapshots:
+    name: "🔍 Check for valid snapshot urls"
+    runs-on: ubuntu-20.04
+    steps:
+
+      - name: Set up Go 1.19
```

Comment:
Do we really need Go here?

Reply:
Hmm. I think you know the answer to that ;)